fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/QueryPlanHashTest.java

```diff
@@ -72,18 +72,13 @@ void testSingleEqualsFilter() {
         // Scan(<,>) | [MySimpleRecord] | num_value_2 EQUALS $3
         RecordQueryPlan plan3 = createPlan("MySimpleRecord", Query.field("num_value_2").equalsParameter("3"));
 
-        assertEquals(913370523, plan1.planHash(PlanHashable.PlanHashKind.LEGACY));
-        assertEquals(913370524, plan2.planHash(PlanHashable.PlanHashKind.LEGACY));
-        assertEquals(913370573, plan3.planHash(PlanHashable.PlanHashKind.LEGACY));
+        assertEquals(913370523, plan1.planHash(PlanHashable.CURRENT_LEGACY));
+        assertEquals(913370524, plan2.planHash(PlanHashable.CURRENT_LEGACY));
+        assertEquals(913370573, plan3.planHash(PlanHashable.CURRENT_LEGACY));
 
-        assertEquals(-1652411480, plan1.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-        assertEquals(-1652411479, plan2.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-        assertEquals(-383271037, plan3.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-
-        assertEquals(448300575, plan1.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
-        assertEquals(448300575, plan2.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
-        // Note that value and parameter comparisons are not the same
-        assertEquals(-383271037, plan3.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
+        assertEquals(1888220942, plan1.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
+        assertEquals(1888220943, plan2.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
+        assertEquals(-1137605911, plan3.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
```

Comment:
A set of changed plan hashes. I take it these are related to the other ones identified. There are a bunch of other such instances in this file

Reply:
These are related to the change in ``PlanHashable` that treats maps and sets different from iterables. That's new stuff. Also, there are some changes related to not hashing certain type information anymore.